plugins/services/src/js/utils/ServiceConfigUtil.js

```diff
@@ -3,10 +3,16 @@ import Networking from '../../../../../src/js/constants/Networking';
 
 const ServiceConfigUtil = {
 
+  matchVIPLabel(str) {
+    return Networking.VIP_LABEL_REGEX.test(str);
+  },
+
+  findVIPLabel(labels = {}) {
```

Comment:
Let's change this util to return a list of VIPs to acknowledge that fact that there can be more than an VIP label.